echidna/core/fill_spectrum.py

```diff
@@ -0,0 +1,65 @@
+import ROOT
+import rat
+import math
+
+def _scint_weights(initial_time, final_time, T):
+    """This method applies to the scintillator backgrounds.
+    It produces the list of time periods to study the time 
+    dependence. For each time period it is calculated the
+    weight, based on radioactive decay rate formula, and
+    stored in a list. Both, studied time and Half-life must
+    be written in the same units. 
+ 
+    Args:
+      initial_time (int): The beginning of time period to study
+      final_time (int): The end time of time period to study
+      T (float): The Half-life of a studied background
+
+    Returns:
+      (*tuple*). Times (*list* of *int*) and Weights (*list* of *float*)
+    """
+    times = [0]
+    weights = [0]
+    for time_step in range(initial_time, final_time):
```

Comment:
I think the step should be `Spectra._time_width` and the initial and final times should always be `Spectra._time_low` and `Spectra._time_high`. Or better still something like:
```python
for time_step in range(0, Spectra._time_bins):
    time = time_step * Spectra._time_width + Spectra._time_low
```